datafusion/core/src/datasource/file_format/write/orchestration.rs

```diff
@@ -173,19 +173,9 @@ pub(crate) async fn stateless_serialize_and_write_files(
 
     // Finalize or abort writers as appropriate
     for mut writer in finished_writers.into_iter() {
-        match any_errors {
-            true => {
-                let abort_result = writer.abort_writer();
-                if abort_result.is_err() {
-                    any_abort_errors = true;
-                }
-            }
-            false => {
-                writer.shutdown()
+        writer.shutdown()
                     .await
                     .map_err(|_| internal_datafusion_err!("Error encountered while finalizing writes! Partial results may have been written to ObjectStore!"))?;
```

Comment:
Don't know whether it's proper to let all just `shutdown` here. But I think according to https://github.com/apache/arrow-datafusion/pull/9648#issuecomment-2002483969, it's OK? 👀